templates/outer-loop/Dockerfile

```diff
@@ -40,19 +40,30 @@ COPY --from=compile --chown=1001:0 /work/configlibdir/ /config
 # 2c) Copy user defined server config, bootstrap.properties, etc.
 COPY --from=compile --chown=1001:0 /work/config/ /config/
 
-# 2d) Install the required features based on the defined configuration. This enables Liberty features and image size to be fit-for-purpose using featureUtility.
-RUN features.sh
-
-# 2e) Add the microprofile health feature configuration if it is not already defined in the user's configuration.
+# 2d) Add the microprofile health feature configuration if it is not already defined in the user's configuration.
+#     This allows k8s to use the deployment's health probes.
 RUN mkdir -p /tmp/stack/config/configDropins/defaults
 COPY --from=compile --chown=1001:0 /stack/ol/config/configDropins/defaults/ /tmp/stack/config/configDropins/defaults/
-RUN if ! /opt/ol/wlp/bin/productInfo featureInfo | grep -q mpHealth-[0-9]*.[0-9]*; then \
-        cp /tmp/stack/config/configDropins/defaults/liberty-stack-mpHealth.xml /config/configDropins/defaults; \
-        features.sh; \
+
+ARG ADD_MP_HEALTH=true
+RUN if [ "$ADD_MP_HEALTH" = "true" ]; then \
+        /opt/ol/wlp/bin/server start; \
+        /opt/ol/wlp/bin/server stop; \
```

Comment:
Is this sequence guaranteed to produce the CWWKF0012I message?   What is the "contract" for `server start` (w.r.t when it returns)?   Might need to investigate and check with the kernel team.

Reply:
Checked with kernel team - start itself _may_ return before the message is printed, but the start/stop combo should be pretty full proof